config/initializers/rack_attack.rb

```diff
@@ -0,0 +1,12 @@
+class Rack::Attack
+  # Rack::Attack.cache.store = ActiveSupport::Cache::MemoryStore.new 
+
+  throttle('api/ip', limit: 1, period: 10) do |req|
```

Comment:
ok i will change it to `throttle('api/ip', limit: 1, period: 10.seconds) do |req|`